modules/@apostrophecms/rich-text-widget/index.js

```diff
@@ -335,8 +335,11 @@ module.exports = {
       optionsToSanitizeHtml(options) {
         return {
           ...sanitizeHtml.defaults,
-          allowedTags: self.toolbarToAllowedTags(options),
-          allowedAttributes: self.toolbarToAllowedAttributes(options),
+          allowedTags: [...self.toolbarToAllowedTags(options), 'pre'],
+          allowedAttributes: {
+            ...self.toolbarToAllowedAttributes(options),
+            'pre': ['*']
```

Comment:
If a change to allowed attributes is needed it should be added to `toolbarToAllowedAttributes` rather than being merged in afterward.

However allowing all attributes is a security issue, and I also don't understand why allowing all attributes would help with line breaks.

Can you clarify the reasoning behind this code?